packages/panw_cortex_xdr/data_stream/alerts/elasticsearch/ingest_pipeline/default.yml

```
@@ -49,6 +49,11 @@ processors:
       formats:
         - UNIX_MS
       if: ctx.panw_cortex?.xdr?.events?.event_timestamp != null
+  - date:
+      field: panw_cortex.xdr.events.agent_host_boot_time
+      formats:
+        - UNIX_MS
+      if: ctx.panw_cortex?.xdr?.events?.agent_host_boot_time != null
```

I see `panw_cortex.xdr.events.event_timestamp` and `panw_cortex.xdr.events.agent_host_boot_time` can both be present. With this change `panw_cortex.xdr.events.agent_host_boot_time` will override `panw_cortex.xdr.events.event_timestamp`. Is that intended ? If so a comment would help. If not -

```suggestion
      if: ctx.panw_cortex?.xdr.events?.event_timestamp == null && ctx.panw_cortex?.xdr?.events?.agent_host_boot_time != null
```

---

Added `target_field` so it wont update `@timestamp`